### PR TITLE
update the indicators context standart

### PR DESCRIPTION
**Pull request by moishce**
## Status
In Progress

## Related Issues
https://github.com/demisto/content/pull/12186

## Description
update the indicators context standard

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/integrations/context-standards-mandatory.md [hunk 1]
```diff
@@ -371,7 +524,34 @@ The following is the format for a Domain. Please note that for WHOIS, the entity
...
+   "DomainReferringIPs": "STRING, ",
+   "DomainReferringSubnets": "STRING, ",
+   "Port": "STRING, ",
+   "Internal": "STRING, ",
+   "Category": "STRING, ",
+   "Campaign": "STRING, ",
+   "TrafficLightProtocol": "STRING, ",
+   "ThreatTypes": "STRING, ",
+   "Tech": {
+       "Country": "STRING, ",
```
Comment (ytauman) [inline docs/integrations/context-standards-mandatory.md:538]: The country of the domain technical contact.

File: docs/integrations/context-standards-mandatory.md [hunk 2]
```diff
@@ -371,7 +524,34 @@ The following is the format for a Domain. Please note that for WHOIS, the entity
...
+       "Name": "STRING, ",
```
Comment (ytauman) [inline docs/integrations/context-standards-mandatory.md:539]: The name of the domain technical contact.

File: docs/integrations/context-standards-mandatory.md [hunk 3]
```diff
@@ -371,7 +524,34 @@ The following is the format for a Domain. Please note that for WHOIS, the entity
...
+       "Organization": "STRING, ",
```
Comment (ytauman) [inline docs/integrations/context-standards-mandatory.md:540]: The organization of the domain technical contact.

File: docs/integrations/context-standards-mandatory.md [hunk 4]
```diff
@@ -371,7 +524,34 @@ The following is the format for a Domain. Please note that for WHOIS, the entity
...
+       "Email": "STRING, "
```
Comment (ytauman) [inline docs/integrations/context-standards-mandatory.md:541]: The email address of the domain technical contact.

File: docs/integrations/context-standards-mandatory.md [hunk 5]
```diff
@@ -371,7 +524,34 @@ The following is the format for a Domain. Please note that for WHOIS, the entity
...
+        "note": "STRING, ",
+        "timestamp": "DATE, ",
+   },
+   "Publications": {
+        "source": "STRING, ",
+        "title": "STRING, ",
+        "link": "STRING, ",
+        "timestamp": "DATE, ",
+   },
+   "Billing": "STRING, ",
```
Comment (ytauman) [inline docs/integrations/context-standards-mandatory.md:553]: The billing address of the domain.

File: docs/integrations/context-standards-mandatory.md [hunk 6]
```diff
@@ -521,7 +761,28 @@ The following is the format for a URL entity.
          "type": "STRING, The type of the indicators that are associated with the URL",
          "description": "STRING, The description of the indicators that are associated with the URL."
     },
-    "MalwareFamily": "STRING, The malware family associated with the URL"
+    "MalwareFamily": "STRING, The malware family associated with the URL",
```
Comment (ytauman) [inline docs/integrations/context-standards-mandatory.md:764]: missing period at the end (like the rest of the descriptions)

File: docs/integrations/context-standards-mandatory.md [hunk 7]
```diff
@@ -521,7 +761,28 @@ The following is the format for a URL entity.
...
+    "Port": "STRING, ",
+    "Internal": "STRING, ",
+    "Campaign": "STRING, ",
+    "TrafficLightProtocol": "STRING, ",
+    "threat_types": "STRING, ",
```
Comment (ytauman) [inline docs/integrations/context-standards-mandatory.md:769]: i think it's supposed to be `ThreatTypes` - no? (like the others have)

File: docs/integrations/context-standards-mandatory.md [hunk 8]
```diff
@@ -521,7 +761,28 @@ The following is the format for a URL entity.
...
+    "ASN": "STRING, ",
```
Comment (ytauman) [inline docs/integrations/context-standards-mandatory.md:770]: The autonomous system name for the URL, for example: 'AS8948'.

File: docs/integrations/context-standards-mandatory.md [hunk 9]
```diff
@@ -521,7 +761,28 @@ The following is the format for a URL entity.
...
+    "ASOwner": "STRING, ",
+    "Geo": {
+      "Country": "STRING, "
```
Comment (ytauman) [inline docs/integrations/context-standards-mandatory.md:773]: The country in which the URL is located.

**Review by ShahafBenYakir — APPROVED** [review]

File: docs/integrations/context-standards-mandatory.md [hunk 10]
```diff
@@ -44,7 +44,31 @@ The following is the format for a file. File here refers to the file indicator o
...
+            "link": "STRING, A link to the original article.",
+            "timestamp": "DATE, The time in which the article was published."
+        }
+        "Behavior": {
+            "details": "STRING, The technical details which are relevant to the behavior.",
+            "title": "STRING, The behavior associated with the file."
+        }
+        "ThreatTypes": {
+            "threatcategory": "STRING, .",
+            "threatcategoryconfidence": "STRING, ."
```
Comment (moishce) [inline docs/integrations/context-standards-mandatory.md:66]: @ytauman I am missing 2 more descriptions
Can you help me with that?

**Review by ShahafBenYakir — COMMENTED** [review]
missing commas before the `"Behavior":` and `"ThreatTypes":`

**Pull request merged**